### PR TITLE
Added Guardians Of The Galaxy

### PR DESCRIPTION
**Pull request by Arkhamfan69**
Adding Guardians Of The Galaxy To The Auto Splitter List On Livesplit

If you are adding or modifying an Auto Splitter, please fill out this checklist:

- [Check ]  My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)

- [Check ] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.

- [Check ] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Review by Arkhamfan69 — COMMENTED** [review]
<Games>
          <Game>Marvel's Guardians of the Galaxy</Game>
</Games>

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Marvel's Guardians of the Galaxy
+        <URLs>
+            <Url>https://raw.githubusercontent.com/Arkhamfan69/My-Splitters-Load-Removers/refs/heads/main/Autosplitters/Gotg%20Splitter.asl
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:7]:
```suggestion
            <URL>https://raw.githubusercontent.com/Arkhamfan69/My-Splitters-Load-Removers/refs/heads/main/Autosplitters/Gotg%20Splitter.asl</URL>
```

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Marvel's Guardians of the Galaxy
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:5]:
```suggestion
            <Game>Marvel's Guardians of the Galaxy</Game>
```

**Comment by Arkhamfan69** [comment]
There is still an issue in the < / Games > not being under the Game Name

**Comment by wooferzfg** [comment]
> There is still an issue in the < / Games > not being under the Game Name

Agreed, please fix that since it's your PR

File: LiveSplit.AutoSplitters.xml [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+<AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Marvel's Guardians of the Galaxy</Game>
+</Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Arkhamfan69/My-Splitters-Load-Removers/refs/heads/main/Autosplitters/Gotg%20Splitter.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description> Autosplitting And Load Remover Made by "Arkhamfan69, TpRedNinja, Kunodemetries"</Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:11]:
```suggestion
    <AutoSplitter>
        <Games>
            <Game>Marvel's Guardians of the Galaxy</Game>
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/Arkhamfan69/My-Splitters-Load-Removers/refs/heads/main/Autosplitters/Gotg%20Splitter.asl</URL>
        </URLs>
        <Type>Script</Type>
        <Description> Autosplitting And Load Remover Made by "Arkhamfan69, TpRedNinja, Kunodemetries"</Description>
    </AutoSplitter>
```

File: LiveSplit.AutoSplitters.xml [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
 <AutoSplitters>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:12]:
```suggestion
<AutoSplitters>
    <AutoSplitter>
        <Games>
            <Game>Marvel's Guardians of the Galaxy</Game>
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/Arkhamfan69/My-Splitters-Load-Removers/refs/heads/main/Autosplitters/Gotg%20Splitter.asl</URL>
        </URLs>
        <Type>Script</Type>
        <Description> Autosplitting And Load Remover Made by "Arkhamfan69, TpRedNinja, Kunodemetries"</Description>
    </AutoSplitter>
```

**Pull request merged**